numba_dppy/examples/debug/simple_dppy_func.py

```diff
@@ -0,0 +1,46 @@
+# Copyright 2020, 2021 Intel Corporation
```

Comment:
Why dppy_func.py is not simple enough?

Reply:
Removed unnecessary functions and comments. Now simple_dppy_func is rather simplified